scripts/deploy.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+rm -rf /indico-client/dist
+pip install twine
+python setup.py sdist
+echo $TWINE_USERNAME
+echo $TWINE_REPOSITORY
+twine upload --repository testpypi /indico-client/dist/*
```

Comment:
assuming you'll sub in $TWINE_REPOSITORY here? or is there a reason not to hardcode it?

Reply:
I'll sub in $TWINE_RESPOSITORY or (more likely) remove it entirely, for now I just wanted to be really sure I didn't push anything anywhere besides testpypi.